tasties_app/tests/test_rating.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from django.contrib.auth.models import User
+from tasties_app.models import Rating, Recipe
+
+
+@pytest.fixture()
+def rating():
+    """
+    This fixture create the data we need in order to execute the tests.
+    """
+    user1 = User.objects.create_user("testuser1", "password")
+    recipe = Recipe(
+        title="Test Recipe1",
+        author_id=user1,
+        description="Test Description1",
+        directions="Test Directions1",
+        minutes_to_make=1,
+        recipe_picture="test_picture2",
+    )
+    recipe.save()
+
+    rating = Rating(author_id=user1, recipe_id=recipe, rating=3)
+    rating.save()
+    return rating
+
+
+class TestRatingModel:
+    @pytest.mark.django_db
+    def test_add_rating(self, rating):
+        assert rating in Rating.objects.all()
+
+    @pytest.mark.django_db
+    def test_delete_rating(self, rating):
+        rating.delete()
+        assert len(Rating.objects.all()) == 0
+
+    @pytest.mark.django_db
+    def test_edit_rating(self, rating):
+        rating_set = Rating.objects.all()
+        rating = rating_set[0]
+        rating.update_rating(4)
+        assert rating.rating == 4
+
+    @pytest.mark.django_db
+    def test_add_bad_rating(self, rating):
+        rating_set = Rating.objects.all()
+        rating = rating_set[0]
+        with pytest.raises(ValueError):
+            rating.update_rating(6)
+
+    @pytest.mark.django_db
+    def test_add_bad_rating_aid(self, rating):
```

Comment:
It's unclear from the name, what is the difference between this test and the other test with the same name?

Reply:
> It's unclear from the name, what is the difference between this test and the other test with the same name?

Added docstring for the functions

Reply:
We usually try to make the test name self-explained. I think you can change the name to something like 
```suggestion
    def test_invalid_rating_author_id(self, rating):
```